docs/DeveloperGuide.md

```diff
@@ -152,7 +152,7 @@ extension feature to this project.<br/><br/>
 The **API** of this component is specified
 in [`Storage.java`](https://github.com/AY2223S2-CS2103T-T13-2/tp/tree/master/src/main/java/seedu/recipe/storage/Storage.java)
 
-Structure of the Storage Component: (tbc)
+![Structure of the Storage Component](images/StorageClassDiagram.png)
```

Comment:
Don't think it appears on the DG site eventually but for neatness should try to follow the `<img src="xxx" width="xxx">` format, and perhaps from there you can adjust appropriately the diagram size(s).